distributed/worker.py

```diff
@@ -1100,6 +1091,22 @@ def _to_dict(self, *, exclude: Container[str] = ()) -> dict:
     # External Services #
     #####################
 
+    def batched_send(self, msg: dict[str, Any]) -> None:
+        """Send a fire-and-forget message to the scheduler through bulk comms.
```

Comment:
```suggestion
        """Send a fire-and-forget message to the scheduler through bulk comms.
        
        If we're not currently connected to the scheduler, the message will be silently dropped!
```
Just seems important to make clear.